drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -126,25 +132,43 @@ struct nrf_usbd_ep_ctx {
 	volatile bool write_in_progress;
 };
 
+/**
+ * @brief Endpoint event structure
+ *
+ * @param ep		Endpoint convtrol block pointer
```

Comment:
Is this the correct documentation of structures? I thought we use Doxygen. This looks more like copy paste of Linux documentation style.

Reply:
@b-gent can you tell?